server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -230,6 +230,15 @@ public Collection<ExecutorBuilder> builders() {
         Setting.Property.NodeScope
     );
 
+    // A setting to change the alpha parameter of the EWMA used in WRITE, SYSTEM_WRITE and SYSTEM_CRITICAL_WRITE thread pools
+    public static final Setting<Double> WRITE_THREAD_POOLS_EWMA_ALPHA_SETTING = Setting.doubleSetting(
```

Comment:
We need to expose the new setting, i.e. add it to `ClusterSettings#BUILT_IN_CLUSTER_SETTINGS`. Otherwise it cannot actually be configured by users. This also suggests we are missing a test for it.

Reply:
🤦🏼‍♂️
Thanks, see https://github.com/elastic/elasticsearch/pull/112283/commits/141d9e09099623599be399ac4bc2781e5229dd5a.